iFixit/react-commerce
# Facets: Convert most to single select

Makes all displayed facets single selects, except for Price Range and Capacity.

## QA

Make sure faceting still functions properly, on desktop and mobile. Make sure the proper links are still rendered for each Item Type facet option. Thanks!

@sterlinghirsh @sumukhipandey 

Closes #374
